modules/tinymce/src/themes/silver/main/ts/ui/icons/Icons.ts

```diff
@@ -74,7 +74,7 @@ const needsRtlTransform = (iconName: string) =>
 const addFocusableBehaviour = () =>
   AddEventsBehaviour.config('add-focusable', [
     AlloyEvents.runOnAttached((comp) => {
-      // set focusable=false on SVGs to prevent IE 11 from focusing the toolbar when tabbing into the editor
+      // set focusable=false on SVGs to prevent focusing the toolbar when tabbing into the editor
       SelectorFind.child(comp.element, 'svg').each((svg) => Attribute.set(svg, 'focusable', 'false'));
```

Comment:
Did we want to remove this as well? It shouldn't be needed on modern browsers but I left it here to be safe :shrug: